test/integration/testsuite_test.go

```diff
@@ -82,19 +82,14 @@ var _ = BeforeSuite(func() {
 	}
 
 	ginkgoOpts = os.Getenv("GINKGO_OPTS")
-	if err != nil {
-
+	if ginkgoOpts == "" {
 		logrus.Infof("Error: Could not read GINKGO_OPTS.")
-		logrus.Infof("%v", err)
 	}
-	Expect(err).NotTo(HaveOccurred())
+	Expect(ginkgoOpts).NotTo(BeEmpty())
```

Comment:
Why can't they be empty?

Reply:
yes, `ginkgoOpts` could be empty. I just checked and found this variable is not used after this assignment.
The environmental variable `GINKGO_OPTS` is only used in makefile L244. 
So I think `ginkgoOpts` part code can be deleted.